tutorials/scripting/cpp/build_system/scons.rst

```diff
@@ -73,3 +73,37 @@ There are two popular ways by which cross platform builds can be achieved:
 `godot-cpp-template <https://github.com/godotengine/godot-cpp-template>`__ contains an
 `example setup <https://github.com/godotengine/godot-cpp-template/tree/main/.github/workflows>`__
 for a GitHub based CI workflow.
+
+Using custom bindings
```

Comment:
I'm not sure how I feel about calling this "custom bindings". To me, a "binding" is like godot-cpp or godot-rust -- the whole library that can interact with Godot. This is more like using a custom API file to build your bindings with

Maybe this should be like:

```suggestion
Using a custom API file
```

... or similar?